scripts/sync_latest_version.sh

```diff
@@ -37,7 +37,9 @@ function uniq_major_minor_filter() {
 function main() {
     local all_release_tags releases doc_versions
     all_release_tags="$(git tag -l v*.*)"
-    releases="$(uniq_major_minor_filter "${all_release_tags}")"
+    # sed trims this list to 10.
+    all_supported_release_tags="$(echo "${all_release_tags}" | sort -Vr | sed '1,10!d')"
```

Comment:
Does this still include the bugfix docs (but exclude them from the overall count. 

I think the goal is that we want to keep docs for the last 10 major.minor release and still keep the major.minor.hotfix builds versions as well.

As an aside I see the ordered list on the website is not sorted properly either.  https://forsetisecurity.org/releases/ :-(

Reply:
I don't quite understand your question, but ere's what you get. As is this is infinitely better than just deleting the versions in the docs folder *after* it built them all.

Prior to the trim
```
git tag -l v*.* |sort -Vr
v2.19.0
v2.18.0
v2.17.0
v2.16.0
v2.15.1
v2.15.0
v2.14.2
v2.14.1
v2.14.0
v2.13.2
v2.13.1
v2.13.0
v2.12.1
v2.12.0
v2.11.2
v2.11.1
v2.11.0
v2.10.1
v2.10.0
v2.9.0
v2.8.0
v2.7.0
v2.6.0
v2.5.0
v2.3.0
v2.2.0
v2.1.0
v2.0.0
v2.0-rc3
v2.0-rc2
v2.0-rc1
v1.1.12
v1.1.11
v1.1.10
v1.1.9
v1.1.8
v1.1.7
v1.1.6
v1.1.5
v1.1.4
v1.1.3
v1.1.2
v1.1.1
v1.0.2
v1.0.1
v1.0
```
After the sed trim
```
git tag -l v*.* |sort -Vr | sed '1,10!d'
v2.19.0
v2.18.0
v2.17.0
v2.16.0
v2.15.1
v2.15.0
v2.14.2
v2.14.1
v2.14.0
v2.13.2
```

Reply:
We currently support 10 feature versions.  So, this should go from `v2.10.0` to `v2.19.0`.  This counts the bug tags as a version.

Reply:
@blueandgold then just use cut.

```
git tag -l v*.* | sort -Vr | cut -d "." -f1-2 | sed '1,10!d'
v2.19
v2.19
v2.18
v2.17
v2.16
v2.15
v2.15
v2.14
v2.14
v2.14
```